README.md

```diff
@@ -40,13 +52,31 @@ Log.Logger = new LoggerConfiguration()
       onBeforeSendArguments => { /*OnBeforeSend: Action<onBeforeSendArguments>*/ })
     .CreateLogger();
 ```
-
 ------
 
-## Step 3 - Configure the JSON configuration file
+### Option 2 - Using the JSON configuration file
+You can initialize Raygun's Serilog Sink inside a Serilog JSON configuration file using the following examples.
+**Minimum setup example:**
```

Comment:
If this is intended to be a kind of heading, we'll need a new line above it, otherwise it looks like:
![image](https://github.com/MindscapeHQ/serilog-sinks-raygun/assets/3094648/e5b6b461-7979-46d5-9a80-2f2850c6ec42)